tests/test_writer.py

```diff
@@ -269,3 +269,25 @@ def test_dump():
 )
 def test_string_needs_quotes(string, expected):
     assert string_needs_quotes(string) is expected
+
+
+def test_single_line_tuples():
+    assert openstep_plist.dumps({"a": 1, "b": (2, 3), "c": "Hello"}, indent=0) == (
+        """{
+a = 1;
+b = (
+2,
+3
+);
+c = Hello;
+}"""
+    )
+    assert openstep_plist.dumps(
+        {"a": 1, "b": (2, 3), "c": "Hello"}, indent=0, single_line_tuples=True
+    ) == (
+        """{
+a = 1;
+b = (2,3);
```

Comment:
I'd have expected to find a space after the comma. But I guess that's what Glyphs does to shave off one byte? Ugh.

Reply:
Kind of makes sense when this is on every node in a file. Separate issue, but Glyphs also writes empty dictionaries without newlines, so that's another thing...

```
E               {
E               pos = 700;
E               },
E               {
E               pos = 500;
E               },
E             - {
E             - },
E             + {},
E             ? +
E               {
E               pos = -200;
E               }
E               );
```

At some point I think we just need to give up and aim for "something Glyphs can read without errors" rather than "100% whitespace-compatible round-tripping".

Reply:
> something Glyphs can read without errors

wait, are you saying that Glyphs trips up with an error when reading a tuple that has spaces after the comma, or an empty dict that is not inlined? that'd be weird.

Reply:
No, I'm saying it doesn't. While it's *nice* to aim for 100% whitespace compatibility, it's not necessary. I guess what I'm saying is when we test round-tripping in glyphsLib test suites we could be more relaxed about whitespace differences.

Reply:
> when we test round-tripping in glyphsLib test suites we could be more relaxed about whitespace differences.

i agree

But in that case, I would care less about 100% whitespace compatibility and more about getting an output that matches reasonable user's expectations, like having a space after the comma in a single-line tuple.